src/Manifest/Manifest.router.ts

```diff
@@ -83,6 +92,26 @@ export class ManifestRouter extends Router {
     }
   }
 
+  async getProjectManifestByCoordinates(req: AuthRequest) {
+    const created_location = server.extractFromReq(req, 'land')
```

Comment:
I'm seeing three different concepts to describe what it seems to be the same thing:
- Coordinates
- land
- created_location

Maybe we could just settle on a name and use that everywhere? coordinates seems like a good name, or maybe just `landId`?

Reply:
Changed for created_location, I think since the database field have this name it will be more correct